rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -344,6 +345,8 @@ void gui_application::InitializeCallbacks()
 	callbacks.on_stop   = [this]() { OnEmulatorStop(); };
 	callbacks.on_ready  = [this]() { OnEmulatorReady(); };
 
+	callbacks.on_missing_fw = [this]() { return OnMissingFwPopUp(); };
```

Comment:
It is wrong to implement the dialog in the main_window, since "no-gui" users won't ever see the error.
Just move the implementation here and pass m_main_window as a parent to the dialog.

Reply:
Done.